01-syntax/01-language-basics/myfigure.js

```diff
@@ -0,0 +1,32 @@
+var moneyTreeSize = 20;
```

Comment:
Few concerns here 
* It won't pass `eslint` 
*  No any description on what type of figure it might be

Provide header with list of global vars which use, for example https://github.com/nesterone/sf-eng-init-gl-spring-2016/blob/master/01-syntax/04-high-order-functions/life-expectancy/lifeExpectancy.js#L1

Also, in header provide a small description of what's your program is doing

Rename file to the name which describes your figure